trunk/source/Library-ModelCheckerUtils/src/de/uni_freiburg/informatik/ultimate/lib/modelcheckerutils/tracehandling/AutomatonFreeRefinementEngine.java

```diff
@@ -126,7 +139,7 @@ private LBool executeStrategy() {
 		}
 
 		// trace was feasible, return
-		if (feasibilityResult == LBool.SAT) {
+		if (feasibilityResult == LBool.SAT || mQuickCheck) {
```

Comment:
This seems wrong -- the log should only be written if the trace is truly feasible.